pkgs/development/tools/bloaty/default.nix

```
@@ -1,23 +1,55 @@
-{ lib, stdenv, cmake, zlib, fetchFromGitHub }:
+{ lib, stdenv, cmake, zlib, fetchFromGitHub, re2, abseil-cpp, protobuf, capstone, gtest, pkg-config, lit, llvmPackages_16 }:
 
 stdenv.mkDerivation rec {
-  version = "1.1";
   pname = "bloaty";
+  version = "unstable-2023-11-06";
```

```suggestion
  version = "1.1-unstable-2023-11-06";
```

[New conventions](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-naming)